src/ds/d500/d500-fw-update-device.cpp

```diff
@@ -143,7 +142,7 @@ ds_d500_update_device::ds_d500_update_device( std::shared_ptr< const device_info
             for (int i = 1; i <= 100; i++)
             {
                 auto percentage_of_transfer = i;
-                auto progress_for_bar = compute_progress(static_cast<float>(percentage_of_transfer), 20.f, 100.f, 5.f) / 100.f;
+                auto progress_for_bar = compute_progress(percentage_of_transfer / 100.f, 20.f, 100.f, 0.f) / 100.f;
```

Comment:
Please check that this change is OK both when working with viewer and rs-fw-update tools.

Reply:
Tested :)